### PR TITLE
Take out Garbage

### PR DESCRIPTION
**Pull request by johaq**
#493

### CODE REVIEW

File: tasks/TakeOutGarbage.tex [hunk 1]
```diff
@@ -1,43 +1,49 @@
-\section{Take Out the Garbage [House Holder]}
```
Comment (kyordhel) [inline tasks/TakeOutGarbage.tex]: Why are we removing the Scenario?

Comment (johaq) [reply tasks/TakeOutGarbage.tex]: Yeah I was at first thinking this is a leftover of when we had this every task as GPSR idea in mind. Are these scenarios mentioned in the general rules? I think they should be if we have them in the title.

Comment (kyordhel) [reply tasks/TakeOutGarbage.tex]: The scenarios are mentioned in §3.6.1 and §3.6.2. Manipulation tasks can only be chosen within *House Holder/Keeper* scenario, while HRI tasks belong to *Party Host*. Test should specify to which scenario they belong.

Comment (johaq) [reply tasks/TakeOutGarbage.tex]: Yeah definitely. I'll add it back in my PRs

**Pull request merged**